src/network/prover.rs

```diff
@@ -236,6 +207,12 @@ impl<N: Network, E: Environment> Prover<N, E> {
     ///
     pub(super) async fn update(&self, request: ProverRequest<N>) {
         match request {
+            ProverRequest::PoolRequest(operator_ip, share_difficulty, block_template) => {
+                // Process the pool request message.
+                self.process_pool_request(operator_ip, share_difficulty, block_template).await;
+                // Proceed to register the prover to receive the next block template.
+                // self.send_pool_register().await;
```

Comment:
Making sure this wasn't missed; if not, this could use a `TODO`

Reply:
It can be deleted - it caused an infinite loop of messages before and I tentatively commented it out to see what would happen.